src/ucp/core/ucp_worker.c

```diff
@@ -3175,8 +3175,8 @@ static int ucp_worker_do_ep_keepalive(ucp_worker_h worker, ucs_time_t now)
     ucs_trace("ep %p: do keepalive on lane[%d]=%p ep->flags=0x%x", ep, lane,
               ep->uct_eps[lane], ep->flags);
 
-    status = ucp_ep_do_uct_ep_keepalive(ep, ep->uct_eps[lane], rsc_index, 0,
-                                        NULL);
+    status = ucp_ep_do_uct_ep_keepalive(ep, ep->uct_eps[lane], lane, rsc_index,
```

Comment:
no need to pass uct_ep, lane and rsc_index to all functions now
we can get them from ep (lane is always keepalive_lane)

Reply:
done